Orange/widgets/model/tests/test_owlogisticregression.py

```diff
@@ -58,3 +57,37 @@ def test_output_coefficients(self):
         self.send_signal("Data", self.data)
         self.widget.apply_button.button.click()
         self.assertIsInstance(self.get_output("Coefficients"), Table)
+
+    def test_domain_with_more_values_than_table(self):
+        """
+        When the data with a domain which has more values than
+        a table was sent, the widget threw error Invalid number of variable columns.
+        GH-2116
+        """
+        data = Table("iris")
+        data = data[:80, :]
+        self.send_signal("Data", data)
+        self.widget.apply_button.button.click()
```

Comment:
This only checks that the widget does not crash. Could you also test that the output is sensible?

Can you try it with keeping the first and the third class, or the second and the third?